README.md

```diff
@@ -1,7 +1,7 @@
 > [!IMPORTANT]  
 > This repository is archived.
 >
-> Check out our [new documentation](https://github.com/fishjam-cloud/documentation) for [Fishjam Cloud](https://fishjam.io/).
+> Check out our [documentation](https://docs.fishjam.io/) for the new [Fishjam](https://fishjam.io/).
```

Comment:
Not sure if we want to link here to documentation or to repo with documentation. If someone is here, I would expect that person looks for some repo with docs, not actual docs. But no strong opinion